### DISCUSSION TITLE
How do I access a model from an ensemble?

### DISCUSSION THREAD

**Discussion by offchan42** [mljar/mljar-supervised#473] [Q&A] [answered]
Suppose I have trained a model using `automl.fit(X, y)` and `automl` is now an ensemble, how do I access `47_LightGBM` for use in prediction?
![image](https://user-images.githubusercontent.com/15215732/136833879-f61ee2c3-a836-4c36-8e22-2fdf160cb910.png)

**Comment by pplonski** [comment, marked as answer]
Hey, sorry right now it is not possible. There is an issue for this https://github.com/mljar/mljar-supervised/issues/423 You can try to manually change the params.json and set `best_model` to selected model.
